halpybot/packages/announcer/announcer.py

```diff
@@ -263,7 +272,11 @@ async def format(self, args: AnnouncerArgs, client: HalpyBOT) -> str:
             try:
                 args["Board_ID"] = await create_case(args, codemap, client)
             except ValueError as val_err:
-                raise AlreadyExistsError("Case Already Exists") from val_err
+                if str(val_err) == "KF Coordinates Improperly Formatted":
+                    raise KFCoordsError from val_err
+                else:
+                    raise AlreadyExistsError("Case Already Exists") from val_err
```

Comment:
casting an exception to a string and then comparing it against a magic string.
This could be avoided by subclassing ValueError and catching that instead.

Reply:
Fixed upstream 45ad91f (#338)